src/game/tuning.h

```diff
@@ -23,6 +23,10 @@ MACRO_TUNING_PARAM(VelrampStart, velramp_start, 550)
 MACRO_TUNING_PARAM(VelrampRange, velramp_range, 2000)
 MACRO_TUNING_PARAM(VelrampCurvature, velramp_curvature, 1.4f)
 
+MACRO_TUNING_PARAM(PlayerElasticityX, player_elasticity_x, 0.0f)
+MACRO_TUNING_PARAM(PlayerElasticityY, player_elasticity_y, 0.0f)
+MACRO_TUNING_PARAM(FlagElasticity, flag_elasticity, 0.5f)
```

Comment:
Why would you not add one for flag x/y if you do so for the player :D?

Reply:
>  I simply want to have more playground with physics

😆 try them out, ground bouncyness is weird and wall bouncyness has much more potential IMO. The flag howerever can **only fall** (in defaul physics) and thus only has bouncyness in Y direction